gobblin-temporal/src/main/java/org/apache/gobblin/temporal/cluster/ServiceBasedAppLauncherWithoutMetrics.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.gobblin.temporal.cluster;
+
+import java.util.Properties;
+
+import org.apache.gobblin.runtime.app.ServiceBasedAppLauncher;
+
+
+/**
+ * {@link ServiceBasedAppLauncher} that does not add metrics service. This is different from deactivating the flag
+ * {@link org.apache.gobblin.configuration.ConfigurationKeys#METRICS_ENABLED_KEY} because this will only not initialize
+ * the metrics service in the cluster manager but still allow for {@link org.apache.gobblin.metrics.GobblinTrackingEvent}
+ */
+public class ServiceBasedAppLauncherWithoutMetrics extends ServiceBasedAppLauncher {
+  public ServiceBasedAppLauncherWithoutMetrics(Properties properties, String appName)
+      throws Exception {
+    super(properties, appName);
+  }
+
+  @Override
+  protected void addMetricsService(Properties properties) {
```

Comment:
GobblinMetrics.isEnabled is an existing gate for enabling metrics. But we do not want to disable all metrics because we need the GTE emission for GaaS. We only want to disable the metrics that are added in the original ServiceBasedAppLauncher

Reply:
An alternative is I could add another flag within the serviceBasedAppLauncher that says "ServiceBasedAppLauncher.disableMetrics". But having multiple metrics enable flags would be pretty confusing IMO